### ISSUE TITLE
TestCase>>skipUnless: wrong argument name

### ISSUE THREAD

**Issue by fxgallego** [dolphinsmalltalk/Dolphin#922]
The code for `TestCase>>skipUnless:` is:
```
skipUnless: aBooleanOrBlock
	" If the assumption in aBooleanOrBlock is not true, abandon the running test
	and mark it as passed. "

	aBooleanOrBlock value ifFalse: [TestSkip signal: 'Assumption in #skipUnless: failed']
``` 
But Boolean doesn't responds to   `#value`

**Comment by blairmcg** [comment]
Maybe it is time to add `value` to `Object` (return self, obviously). On the one hand I don't like it because it hides a lot of errors, but on the other hand it is attractive for these kind of code patterns, and every other Smalltalk has done it (I think).

**Comment by fxgallego** [comment]
Pharo and Cuis have `Object>>value`.

We didn't added it after so many years, maybe being a bit cautious. But I added it and ran my tests. It broke one test on an object with #doesNotUnderstand: overridden to access a private hash by sending messages. The hash was expected to contain `value` so the developer sent `value` and it started to fail after I added `value` in Object. Maybe this object should inherit from ProtoObject or process their values in other way but it's an example of what could go wrong.
